scripts/irods/test/test_iadmin.py

```diff
@@ -730,6 +730,48 @@ def test_rebalance_for_repl_node(self):
         self.admin.assert_icommand("iadmin rmresc repl")
         self.admin.assert_icommand("iadmin rmresc pt")
 
+    def test_rebalance_for_tree_with_many_resources__issue_5110(self):
+        hostname = lib.get_hostname()
+
+        try:
+            # =-=-=-=-=-=-=-
+            # STANDUP
+            self.admin.assert_icommand("iadmin mkresc repl replication", 'STDOUT_SINGLELINE', "Creating")
+
+            self.admin.assert_icommand("iadmin mkresc red random", 'STDOUT_SINGLELINE', "Creating")
+            self.admin.assert_icommand("iadmin mkresc green random", 'STDOUT_SINGLELINE', "Creating")
+
+            for i in range(1,301):
+                self.admin.assert_icommand("iadmin mkresc ufs{0} unixfilesystem {1}:/tmp/irods/ufs{0}Vault".format(i,hostname), 'STDOUT_SINGLELINE', "Creating")
```

Comment:
i believe this can be replace with `lib.create_ufs_resource(...)` assuming the vault path is not used anywhere in this test.

Reply:
nice.

Reply:
done